tests/test_methods_and_attributes.py

```diff
@@ -171,6 +171,19 @@ def test_static_properties():
     assert m.TestPropertiesOverride().def_readonly == 99
     assert m.TestPropertiesOverride.def_readonly_static == 99
 
+    # Static attributes can be deleted
```

Comment:
`# Only static attributes can be deleted`

(When I look at this a "but" or "only" is clearly missing, somewhere.)